app-antivirus/clamav/clamav-1.2.0.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="An open source antivirus engine"
+DESCRIPTION="ClamAV is an open source antivirus engine for detecting  \
+trojans, viruses, malware & other malicious threats."
+HOMEPAGE="https://www.clamav.net"
+COPYRIGHT="2001-2007 Tomasz Kojm
+	2007-2023 Cisco Systems"
+PACKAGER="Luca D'Amico <damico.luca91@live.it>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/Cisco-Talos/clamav/archive/refs/tags/clamav-$portVersion.tar.gz"
+CHECKSUM_SHA256="efe7e234fe29a96bf4da23336e38ea2989ed0be162342993e99a11901524f43f"
+SOURCE_DIR="clamav-clamav-$portVersion"
+PATCHES="haiku-$portVersion.patch"
+ADDITIONAL_FILES="
+	clamd.conf
+	freshclam.conf
+	"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/clamav/clamd.conf keep-old
+	settings/clamav/freshclam.conf keep-old
+	"
+
+PROVIDES="
+	clamav$secondaryArchSuffix = $portVersion
+	cmd:clamav_config$secondaryArchSuffix = $portVersion
+	cmd:clambc$secondaryArchSuffix = $portVersion
+	cmd:clamconf$secondaryArchSuffix = $portVersion
+	cmd:clamd$secondaryArchSuffix = $portVersion
+	cmd:clamdscan$secondaryArchSuffix = $portVersion
+	cmd:clamdtop$secondaryArchSuffix = $portVersion
+	cmd:clamscan$secondaryArchSuffix = $portVersion
+	cmd:clamsubmit$secondaryArchSuffix = $portVersion
+	cmd:freshclam$secondaryArchSuffix = $portVersion
+	cmd:sigtool$secondaryArchSuffix = $portVersion
+	lib:libclamav$secondaryArchSuffix = $portVersion
+	lib:libclammspack$secondaryArchSuffix = $portVersion
+	lib:libclamunrar$secondaryArchSuffix = $portVersion
+	lib:libclamunrar_iface$secondaryArchSuffix = $portVersion
+	lib:libfreshclam$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	json_c$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libpcre2_8$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcheck$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libjson_c$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
```

Comment:
You can add `devel:libmspack$secondaryArchSuffix` in here too (and `lib:libmspack$secondaryArchSuffix `in REQUIRES).
Then enable system one with `-DENABLE_EXTERNAL_MSPACK=ON` in BUILD.
This will skip the build for `lib:libclammspack$secondaryArchSuffix`.

Reply:
But then I should remove ` lib:libclammspack$secondaryArchSuffix = $portVersion`  from PROVIDES, right?

Reply:
Right, no need for it anymore as it will use the external one.

Reply:
"Nicked" from Arch Linux :) https://gitlab.archlinux.org/archlinux/packaging/packages/clamav/-/blob/main/PKGBUILD?ref_type=heads#L62